rgbd/src/rgbd_to_shm.cpp

```diff
@@ -42,7 +42,7 @@ int main(int argc, char **argv)
     client.initialize(server_name);
     server.initialize(server_name);
 
-    ros::NodeHandle nh;
+    ros::NodeHandle nh = ros::NodeHandle();
```

Comment:
Don't need this right? The default constructor does the same

Reply:
Indeed not needed.
```suggestion
    ros::NodeHandle nh;
```